docroot/themes/custom/uids_base/scss/components/banner.scss

```diff
@@ -134,14 +152,10 @@
     }
   }
 
-// Set larger heading size for large banner only
-.banner--large,
-.layout--title.banner {
-  .bold-headline {
-    @include breakpoint(md) {
-      font-size: calc(100% + 3.5vw);
-    }
-  }
+// Set large heading size for page titles in banner
+.layout--title.banner .bold-headline {
```

Comment:
Should this be `.headline` instead of `.bold-headline`?

Reply:
Good question.  Let me check that out.